### DISCUSSION TITLE
Windows Event Log Shipping Questions

### DISCUSSION THREAD

**Discussion by kl3ss** [Security-Onion-Solutions/securityonion#5275] [General] [answered]
Hey there,

In regards to shipping of Windows Event Logs into Security Onion, I have the following questions:

Out of Winlogbeats, Osquery and Wazuh Agents, which of these will be automatically be parsed and normalized into ECS/ELK?

Also does Winlogbearts, Osquery and Wazuh support ETW Logs?

Cheers

kl3ss

**Comment by defensivedepth** [comment, marked as answer]
All three agents can ship + SO will parse and ingest WEL, including Sysmon logs.
